src/components/pages/dashboard.tsx

```diff
@@ -471,7 +471,8 @@ const getServerSideProps = getApplicationServerSideProps(
       };
     }
 
-    await Promise.allSettled(
+    await Promise.all(
+      // @ts-expect-error
```

Comment:
why was this expect-error needed?

Reply:
Because it was throwing a TS error? 😅 (prefetch / normal fetch)